ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -156,7 +157,7 @@ tests:
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-gcp-ipi-proxy-private
 - as: ibmcloud-ipi-proxy-private-rt-f14
```

Comment:
Do we need to update the frequency to f28?

Reply:
I will sync other platforms to update the frequency to avoid making the PR too big.